sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -49,6 +49,12 @@ SIREPO.app.directive('advancedEditorPane', function(appState, panelState) {
             var viewInfo = appState.viewInfo($scope.viewName);
             var i;
 
+            function camelToSnakeCase(v) {
+                return v.replace(/[\w]([A-Z])/g, function(m) {
+                    return m[0] + "_" + m[1];
```

Comment:
Does angular accept underscore as well as dashes?

Reply:
Yes, I guess it does. This was unintentional. I think I wanted camelToKebobCase() but I copied the wrong function.